### DISCUSSION TITLE
No new Battery

### DISCUSSION THREAD

**Discussion by ak68-hub** [mr-manuel/venus-os_dbus-mqtt-battery#21] [General]
Hallo together,

unfortunately I´m not able to sse a new device on a Cerbo GX FW 3.3. (old GUI !).

Connection seems to be correct:

root@einstein:~# **svstat /service/dbus-mqtt-battery**
/service/dbus-mqtt-battery: up (pid 2890) 38 seconds

**tail -n 100 -f /data/log/dbus-mqtt-battery/current | tai64nlocal**
2024-04-19 13:43:00.773871500 *** starting dbus-mqtt-battery ***
2024-04-19 13:44:02.701158500 WARNING:root:Waiting since 60 seconds for receiving first data...
2024-04-19 13:44:02.702164500 ERROR:root:Driver stopped. Timeout of 60 seconds exceeded, since no new MQTT message was received in this time.

but ... no battery-device will be recognized in VenusOS

What can I do ? I also tried id with GUI 2 (Beta FW 3.41)- also without succed.

Could be the change of the MQTT-Broker (Mosquitto -> FlashMQ) be source of this problem ?

Thank you
Andreas

**Comment by mr-manuel** [comment]
As the driver say, he gets no new MQTT message. Try to send one manually and check with MQTT Explorer.

**Reply by broekieman** [reply]
How do i send manually a MQTT message. I have MQTT Explorer running.

**Comment by mr-manuel** [comment]
Check the changelog of this post https://community.victronenergy.com/questions/277965/venus-os-v3313.html

**Comment by ak68-hub** [comment]
Thank you Manuel,
I´ve gut the MQTtopics now (with MQTT-Explorer), but there is no device:  /emphase/battery in Cerbo GX
I will try it with Version 3.31-4 and come back again :)
Andreas

**Comment by ak68-hub** [comment]
after this:
mosquitto_sub -d -t '#' -h **hostname** -u '' -P '' -V 5

all Topics can read/ written to Cerbo,

BUT:  there ist **NO device** in Cerbo GX

Even If I send topics to the battery (Power/ Voiltage/ SoC), no device is created on Cerbo GX :(
... or any dates seen in the Display-Scheme overview

I tried dbus-mqtt-solar-charger as well,
THERE the PV is shown immediately as device and at the overwiew-display

Is there a conflict with a Multiplus ? But If I unplug the VE.Bus Multiplus -> same problem

**Reply by mr-manuel** [reply]
1. Did you change the logging to DEBUG?
2. How does your config.ini look like?
3. How does the payload and topic look like you are sending? Best would be a screenshot of MQTT explorer that shows the send payload and topic.

Since many are using this driver, I don't think there is a problem with Venus OS.

**Comment by ak68-hub** [comment]
HI Manuel,
I was successful now, the mistake was a wrong main-topic; **/enphase/battery** instead of **enphase/battery** :( 
Thank you
Andreas
